src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,363 @@
+            }
+
+            public static TensorFlowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema schema)
+            {
+                Contracts.CheckValue(env, nameof(env));
+                env.CheckValue(ctx, nameof(ctx));
+                ctx.CheckAtModel(GetVersionInfo());
+
+                var numInputs = ctx.Reader.ReadInt32();
+                Contracts.CheckDecode(numInputs > 0);
+
+                string[] source = new string[numInputs];
+                for (int j = 0; j < source.Length; j++)
+                    source[j] = ctx.LoadNonEmptyString();
+
+                byte[] data = null;
+                if (!ctx.TryLoadBinaryStream("TFModel", r => data = r.ReadByteArray()))
+                    throw env.ExceptDecode();
+
+                var outputColName = ctx.LoadNonEmptyString();
+
+                return new TensorFlowMapper(env, schema, data, source, outputColName);
+            }
+
+            public void Save(ModelSaveContext ctx)
+            {
+                _host.AssertValue(ctx);
+                ctx.CheckAtModel();
+                ctx.SetVersionInfo(GetVersionInfo());
+
+                var buffer = new TFBuffer();
+                _session.Graph.ToGraphDef(buffer);
+
+                ctx.SaveBinaryStream("TFModel", w =>
+                {
+                    w.WriteByteArray(buffer.ToArray());
+                });
+                Contracts.AssertNonEmpty(InputColNames);
+                ctx.Writer.Write(InputColNames.Length);
+                foreach (var colName in InputColNames)
+                    ctx.SaveNonEmptyString(colName);
+
+                ctx.SaveNonEmptyString(OutputColName);
+            }
+
+            private TFSession LoadTFSession(byte[] modelBytes, string modelArg)
+            {
+                var graph = new TFGraph();
+                try
+                {
+                    graph.Import(modelBytes, "");
+                }
+                catch (Exception ex)
+                {
+                    if (!string.IsNullOrEmpty(modelArg))
+                        throw _host.Except($"TensorFlow exception triggered while loading model from '{modelArg}'");
+#pragma warning disable MSML_NoMessagesForLoadContext
+                    throw _host.ExceptDecode(ex, "Tensorflow exception triggered while loading model.");
+#pragma warning restore MSML_NoMessagesForLoadContext
+
+                }
+                return new TFSession(graph);
+            }
+
+            private ITensorValueGetter CreateTensorValueGetter<T>(IRow input, bool isVector, int colIndex, TFShape tfShape)
+            {
+                if (isVector)
+                    return new TensorValueGetterVec<T>(input, colIndex, tfShape);
+                else
+                    return new TensorValueGetter<T>(input, colIndex);
+            }
+
+            private ITensorValueGetter CreateTensorValueGetterVec(IRow input, TFDataType tfType, bool isVector, int colIndex, TFShape tfShape)
```

Comment:
>Vec [](start = 62, length = 3)

We can get rid of this suffix, since there is no other CreateTensorValueGetter method.